include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -317,8 +317,20 @@ class CollectiveEpilogue<
             make_coord(m_offset, n_offset, l_offset),
             make_shape(_, Int<FragsM>{}, Int<FragsN>{}));
 
-    Tensor mD_crd = make_identity_tensor(make_shape(M,N));
-    Tensor cD = local_tile(mD_crd, take<0,2>(SubgroupTileShape{}), make_coord(m_coord, n_coord));
+    // Because Sm90 uses shared memory, they are not tied to using the same accumulator values
```

Comment:
It shouldn't matter whether a tensor is register or in SLM. If the layout is correct it should point to the correct values. Is the layout not correct before this? If so why?